package.json

```diff
@@ -45,7 +48,7 @@
         "jest-environment-jsdom": "^29.7.0",
         "jest-svg-transformer": "^1.0.0",
         "react-test-renderer": "^18.2.0",
-        "vite": "^5.2.0"
+        "vite": "^5.2.9"
     },
     "jest": {
```

Comment:
can you add test coverage settings to jest please? we can set %85 coverage for all right now

Reply:
vite downgraded to be supported by cypress